cpp/dolfin/mesh/MeshEntity.h

```diff
@@ -125,6 +125,9 @@ class MeshEntity
   /// dimension.
   inline std::size_t num_entities(std::size_t dim) const
   {
+    if (dim == _dim)
+      return 1;
+
```

Comment:
could you add an `else` in?

Reply:
I could do - does it make it better? Some compilers will complain that "control reaches end of function returning void" or something...